### ISSUE TITLE
Failure to install on Ubuntu 22.04

### ISSUE THREAD

**Issue by rpsene** [asciidoctor/asciidoctor-mathematical#122]
When trying to install on a ubuntu 22.04 container image I'm getting 

```
gem update --system; gem install asciidoctor-mathematical          1054.8s
 => => # installed?'
 => => # from /usr/lib/ruby/vendor_ruby/rubygems/request_set.rb:173:in `block (2 levels)
 => => #  in install'
 => => # ERROR:  While executing gem ... (Gem::RemoteFetcher::FetchError)
 => => #     bad response Forbidden 403 (https://index.rubygems.org/quick/Marshal.4.8/ma
 => => # thematical-1.6.16.gemspec.rz)
```

When we look at https://index.rubygems.org/gems/asciidoctor-mathematical there is no version 1.6.16. 

Is this a known issue as I started to see this behavior only last week?

**Comment by rpsene** [comment]
Setting a hard-code dependency works! In this case mathematical:1.6.14

**Comment by bwklein** [comment]
> Setting a hard-code dependency works! In this case mathematical:1.6.14

@rpsene how did you do this from the install side?  

Currently, I am installing gems in a Docker container like this...

```
RUN gem install --no-document \
  "asciidoctor:${ASCIIDOCTOR_VERSION}" \
  asciidoctor-bibtex \
  asciidoctor-confluence \
  asciidoctor-diagram \
  asciidoctor-mathematical \
  asciimath \
  "asciidoctor-pdf:${ASCIIDOCTOR_PDF_VERSION}" \
  asciidoctor-revealjs \
  asciidoctor-html5s \
  coderay \
  haml \
  pygments.rb \
  prawn-gmagick \
  rake \
  rghost \
  rouge \
  slim \
  thread_safe \
  tilt
```

Is there a way from this command to tell asciidoctor-mathematical to install 1.6.14 of the mathematical gem it depends on and not the latest release that was 'yanked'? https://rubygems.org/gems/mathematical/versions/1.6.16

I still don't understand how a 'yanked' gem would still try to install as a dependency, but I know very little about gems and how it all works under the hood.

**Comment by rpsene** [comment]
@bwklein yes, take a look here https://github.com/riscv/riscv-docs-base-container-image/blob/main/Dockerfiles/ubuntu2204#L42.

**Comment by bwklein** [comment]
@rpsene That did the trick!  Thank you! 🎉

**Comment by 4e554c4c** [comment]
https://rubygems.org/gems/mathematical/versions/1.6.17 has been released
